presto-spi/src/main/java/com/facebook/presto/spi/type/Chars.java

```diff
@@ -66,4 +66,42 @@ private static int sliceLengthWithoutTrailingSpaces(Slice slice)
         }
         return 0;
     }
+
+    public static int compareChars(Slice left, Slice right)
+    {
+        if (left.length() < right.length()) {
+            return compareCharsShorterToLonger(left, right);
+        }
+        else {
+            return -compareCharsShorterToLonger(right, left);
+        }
+    }
+
+    private static int compareCharsShorterToLonger(Slice shorter, Slice longer)
+    {
+        for (int i = 0; i < shorter.length(); ++i) {
+            int result = compareUnsignedBytes(shorter.getByte(i), longer.getByte(i));
```

Comment:
This will work only for `ANSI` characters. We should actually compare codepoints. But as it is the same behavior as varchar has i think this is fine for now.